pkg/operator/staticpod/controller/prune/prune_controller.go

```diff
@@ -102,7 +103,7 @@ func getRevisionLimits(operatorSpec *operatorv1.StaticPodOperatorSpec) (int32, i
 }
 
 func (c *PruneController) excludedRevisionHistory(operatorStatus *operatorv1.StaticPodOperatorStatus, failedRevisionLimit, succeededRevisionLimit int32) ([]int, error) {
-	var succeededRevisionIDs, failedRevisionIDs, inProgressRevisionIDs, unknownStatusRevisionIDs []int
+	var succeededRevisions, failedRevisions, inProgressRevisions, unknownStatusRevisions []int
```

Comment:
as much as I prefer this of the ID everywhere, please split those refactorings and the actual logic change into two commits. It's hard to review now.